mod.ts

```diff
@@ -226,10 +311,51 @@ export class RhumRunner {
    *       });
    *     });
    */
-  public skip(name: string, cb: () => void): void {
-    // TODO(ebebbington|crookse) Maybe we could still call run, but pass in {
-    // ignore: true } which the Deno.Test will use? just so it displays ignored
-    // in the console
+  public async skip(name: string, cb: () => void): Promise<void> {
+    // If there is no current test suite, then we know we know this .skip() call
+    // is attached to a test suite
+    if (this.getCurrentTestSuite() == "") {
+      this.skipTestSuite(name, cb);
+      // Otherwise, if there is a current test suite, then we know this .skip()
+      // call is attached to a test case
+    } else {
+      // If all of the test cases in a test suite have been set up, then this
+      // .skip() call is for a test suite. Soooo recursion...
+      if (this.getCurrentTestSuiteNumTestCases() <= 0) {
+        this.setCurrentTestSuite("");
+        this.skip(name, cb);
+        return;
+      }
+
+      this.skipTestCase(name, cb);
+    }
+  }
+
+  /**
+   * Skip a test case.
+   *
+   * @param name - The name of the test case.
+   * @param testFn - The test to execute.
+   */
+  public async skipTestCase(name: string, testFn: () => void): Promise<void> {
+    this.addTestCaseToTestSuite(name, testFn, true, this.getCurrentTestSuite());
+  }
```

Comment:
Async fn but doesn't seem to do anything async?

Reply:
15f7b796cdf